ui/pages/notifications/snap/utils/utils.ts

```diff
@@ -0,0 +1,17 @@
+import type { RawSnapNotification, SnapNotification } from '../types/types';
+import { SNAP } from '../types/types';
+
+export const processSnapNotifications = (
```

Comment:
This is an internal process to convert snap notifications into the new shape (which vibes with the rest of out notifications).